### DISCUSSION TITLE
error on t-swap uint64.max

### DISCUSSION THREAD

**Discussion by YavorJJ** [Cyfrin/security-and-auditing-full-course-s23#219] [Q&A]
Hello, I cant figure out what Patrick actually means on Lesson 25, Debugging The Fuzzer, min: 5:55
it is giving him an error while my test is passing successfully without me changing anything like he does in the video with the uint64.max so he can fix it.
Whats exactly is the problem with this uint, and should I change it or continue?

**Comment by EngrPips** [comment]
Hello @YavorJJ, I don't understand what you mean by it is giving you an error but your test is passing.

**Reply by YavorJJ** [reply]
My bad for any confusion earlier. Patrick encountered an error related to uint64.max, which caused him to remove it and modify his code. My tests are passing without any issues related to uint64.max, so my question is whether I should also remove uint64.max like Patrick did, despite not experiencing any errors. Also, I can’t understand WHY Patrick removed uint64.max from his code. Why did he make that change?

**Reply by YavorJJ** [reply]
> Hello @YavorJJ, I don't understand what you mean by it is giving you an error but your test is passing.

Hah, sorry for wasting your time, I just figured it all out.

**Reply by EngrPips** [reply]
Awesome, Keep crushing it, buddy.
